built_value/lib/built_value.dart

```diff
@@ -10,6 +10,8 @@ import 'package:meta/meta_meta.dart';
 /// provide the rest of the implementation.
 ///
 /// See https://github.com/google/built_value.dart/tree/master/example
+/// Intro doc:
+/// https://medium.com/dartlang/darts-built-collection-for-immutable-collections-db662f705eff
```

Comment:
Hmm, that article is for built_collection, not built_value.

But it wouldn't hurt to link to some docs; maybe to the main package doc which links lots of articles, WDYT?

https://pub.dev/packages/built_value

Reply:
Internal to goog: Reviewing cl/441861701 , https://critique.corp.google.com/cl/441861701/depot/google3/assistant/verticals/productivity/portable/fulfillment/frames/lib/delete_cross_device_timer_frame.dart
I'm a complete newb to dart.  I see below code line: Iterable<Frame> build(BuildContext context) {
No idea what is build.  I did a moma and or code search, which lead me to built_value.dart .  Probably based on CS or moma search rank bias.
I look at the code, go to example, doesn't help me understand what is build.
I climb the github source tree and start to get an understanding.  I google the subject which leads me the medium article.
I was thinking rather than others going thru this pain, add doc to the file I found.
If this is not desired then yes maybe add something to main package doc.  I can drop this pull request in that case.

Reply:
Oh. But `build(BuildContext` is a different thing entirely, not related to either built_collection or built_value :)

I think let's close this, since there's no improvement to be made here ;) thanks.